docs/news.rst

```diff
@@ -726,151 +750,159 @@ Example output:
 
-A tag has `name`, `input` file name, and `pattern` as basic information.
-Some fields like `language:`, `signature:`, etc are attached
-to the tag as optional information.
+A tag has a `name`, an `input` file name, and a `pattern` as basic
+information. Some fields like `language:`, `signature:`, etc are
+attached to the tag as optional information.
 
-In exuberant-ctags, fields are common in all languages.
-universal-ctags extends the concept of fields; a parser can define its
-own field. This extension is proposed by @pragmaware in #857.
+In exuberant-ctags, fields are common to all languages.
+universal-ctags extends the concept of fields; a parser can define
+its own field. This extension was proposed by @pragmaware in #857.
 
-For implementing the parser own fields, the option for listing and
+For implementing the parser own fields, the options for listing and
 enabling/disabling fields are also extended.
 
-In ``--list-fields`` output, the owner of the field is printed at `LANGUAGE`
-column:
+In the output of ``--list-fields``, the owner of the field is printed
+in the `LANGUAGE` column:
 
 .. code-block:: console
 
 	$ ./ctags --list-fields
-	#LETTER NAME            ENABLED LANGUAGE        XFMTCHAR DESCRIPTION
+	#LETTER NAME            ENABLED LANGUAGE         XFMTCHAR DESCRIPTION
 	...
-	-       end             off     C               TRUE     end lines of various constructs
-	-       properties      off     C               TRUE     properties (static, inline, mutable,...)
-	-       end             off     C++             TRUE     end lines of various constructs
-	-       template        off     C++             TRUE     template parameters
-	-       captures        off     C++             TRUE     lambda capture list
-	-       properties      off     C++             TRUE     properties (static, virtual, inline, mutable,...)
+	-       end             off     C                TRUE     end lines of various constructs
+	-       properties      off     C                TRUE     properties (static, inline, mutable,...)
+	-       end             off     C++              TRUE     end lines of various constructs
+	-       template        off     C++              TRUE     template parameters
+	-       captures        off     C++              TRUE     lambda capture list
+	-       properties      off     C++              TRUE     properties (static, virtual, inline, mutable,...)
 	-       sectionMarker   off     reStructuredText TRUE     character used for declaring section
-	-       version         off     Maven2          TRUE     version of artifact
+	-       version         off     Maven2           TRUE     version of artifact
 
-e.g. `reStructuredText` is the owner of `sectionMarker` field. Like
-`end` field owned by `C` and `C++`, more than one parsers have fields
-with the same name.
+e.g. `reStructuredText` is the owner of the `sectionMarker` field.
+Like the `end` field is owned by `C` and `C++`, more than one parser
```

Comment:
> Yes, if you remove it then it reads as "Like the end field (owned by C and C++), more than one parser..." shifting the subject to the field rather than the concept of field ownership.

Oh, I see. Thank you very much. 

> It's still pretty awkward though so how about just simplifying it to something like: "e.g. reStructuredText is the owner of the sectionMarker field and both C and C++ own the end field."

Let's use your sentence.

> While it's not enforced I assume it is expected that fields with the same name will be used for the same purpose across languages?

Yes. However, if it is used for the same purpose, we should move the field to the main part.
The parser own field is still very new concept.  So I'm not sure what kind of fields will be introduced/proposed in the future.